contracts/cloud-diff.yml

```
@@ -873,6 +861,1982 @@ paths:
+                                                          - trackWidth
+                                                          - speed
+                                                          - randomColors
+                                                          - trackPointVisualization
+                                                    properties:
+                                                      trackWidth:
+                                                        description: Width of the track
+                                                        type: integer
+                                                      speed:
+                                                        description: Speed of the track animation
+                                                        type: integer
+                                                      randomColors:
+                                                        description: Assign different colors to different tracks
+                                                        type: boolean
+                                                      colors:
+                                                        description: Colors define color encoding of data into a visualization
+                                                        type: array
+                                                        items:
+                                                          $ref: '#/paths/~1dashboards/post/responses/201/content/application~1json/schema/oneOf/1/allOf/1/properties/cells/items/allOf/1/properties/properties/oneOf/0/properties/colors/items'
+                          labels:
+                            $ref: '#/components/schemas/Variable/properties/labels'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/responses/ServerError/content/application~1json/schema'
+    get:
+      operationId: GetDashboards
+      tags:
+        - Dashboards
+      summary: List all dashboards
+      parameters:
+        - $ref: '#/paths/~1users/get/parameters/0'
+        - in: query
+          name: offset
+          required: false
+          schema:
+            type: integer
+            minimum: 0
+        - in: query
+          name: descending
+          required: false
+          schema:
+            type: boolean
+            default: false
+        - in: query
+          name: limit
+          schema:
+            type: integer
+            minimum: -1
+            maximum: 100
+            default: 20
+          description: The non-zero number of dashboards to return
+        - in: query
+          name: owner
+          description: A user identifier. Returns only dashboards where this user has the `owner` role.
+          schema:
+            type: string
+        - in: query
+          name: type
+          description: 'Type of task, unset by default.'
```

```suggestion
          description: 'Type of dashboard, unset by default.'
```

i'm pretty sure this is a generated file, but just to be safe :)